partials/tabs/faq.html

```diff
@@ -14,6 +14,7 @@
 <script>
 $(document).ready(function() {
   $('.collapsible').collapsible();
-});
+    $('ul.tabs').tabs();
 
```

Comment:
The indentation is not correct.

Reply:
but then why is this showing @li-boxuan :sweat_smile: 
```
$ docker run -v=$(pwd):/app --workdir=/app coala/base coala --ci
Unable to find image 'coala/base:latest' locally
latest: Pulling from coala/base
Status: Downloaded newer image for coala/base:latest
Executing section global...
[ERROR][06:03:24] The bear InvalidLinkBear does not fulfill all requirements. You are not connected to the internet.
Executing section commit...
Executing section linecount...
Executing section filenames...
Executing section markdown_filenames...
Executing section spacing...
|    | [NORMAL] SpaceConsistencyBear:
|    | Line contains following spacing inconsistencies:
|    | - No newline at EOF.
|----|    | /app/partials/tabs/faq.html
|    |++++| /app/partials/tabs/faq.html
|  17|  17|       $('ul.tabs').tabs();
|  18|  18| 
|  19|  19|   });
|  20|    |-</script>
|    |  20|+</script>
[INFO][06:03:26] Applied 'ShowPatchAction' on 'partials/tabs/faq.html' from 'SpaceConsistencyBear'.
Executing section html...
Executing section yml...
Executing section new_linelength_projects...
Executing section old_linelength_projects...
Executing section cli...
The command "docker run -v=$(pwd):/app --workdir=/app coala/base coala --ci" exited with 1.
cache.2
store build cache
$ bundle clean
0.00s1.42snothing changed, not updating cache
Done. Your build exited with 1.
```